core/src/main/scala/org/apache/spark/storage/DiskBlockManager.scala

```diff
@@ -40,7 +42,7 @@ private[spark] class DiskBlockManager(conf: SparkConf, deleteFilesOnStop: Boolea
   /* Create one local directory for each path mentioned in spark.local.dir; then, inside this
    * directory, create multiple subdirectories that we will hash files into, in order to avoid
    * having really large inodes at the top level. */
-  private[spark] val localDirs: Array[File] = createLocalDirs(conf)
+  private[spark] val localDirs: Array[File] = createLocalDirs()
```

Comment:
Nit: this change seems unrelated to push-based shuffle so should we not make it?